charts/scalardb-cluster/templates/scalardb-cluster/deployment.yaml

```diff
@@ -52,6 +52,10 @@ spec:
             value: {{ include "scalardb-cluster.fullname" . }}-headless
           - name: SCALAR_DB_CLUSTER_LOG_LEVEL
             value: "{{ .Values.scalardbCluster.logLevel }}"
+          {{- if and (.Values.scalardbCluster.tde.enabled) (eq .Values.scalardbCluster.tde.type "self") }}
+          - name: SCALAR_DB_CLUSTER_ENCRYPTION_SELF_KUBERNETES_SECRET_NAMESPACE_NAME
+            value: {{ .Release.Namespace }}
+          {{- end }}
```

Comment:
If we use the `self` type TDE, we should set the namespace name automatically via an environment variable. This is because ScalarDB Cluster uses this information to run the Kubernetes API to create, get, and delete the secret resources to manage the data encryption key.